### DISCUSSION TITLE
Getting a crash in this patch and really not sure why

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#442] [! Welcome - readme first!]
added you to a patch called blog3.1

when you hit the red menu button so it goes like this
<img width="815" alt="Screenshot 2022-12-27 at 09 16 44" src="https://user-images.githubusercontent.com/36418408/209635400-dfff30f4-7145-4447-9f09-5ccba1ba1f56.png">
then you hit a number to expand the menu so it goes like this
<img width="817" alt="Screenshot 2022-12-27 at 09 16 53" src="https://user-images.githubusercontent.com/36418408/209635433-5d24c3cc-9a07-4d22-8c40-34a280d942dc.png">

I scroll up and down a few times and leave it for a minute or 2 with the menu expanded.
then something crashes. Happens every time on my M1 macbook pro in chrome.

heres the log

RuntimeError: Aborted(OOM). Build with -s ASSERTIONS=1 for more info.
    at qa (https://cables.gl/api/lib/ammo.wasm.js?nc=1671443420321:18:247)
    at d (https://cables.gl/api/lib/ammo.wasm.js?nc=1671443420321:24:371)
    at wasm://wasm/002d84e2:wasm-function[103]:0xa2bf
    at wasm://wasm/002d84e2:wasm-function[317]:0x1a9f1
    at wasm://wasm/002d84e2:wasm-function[1304]:0x6d188
    at wasm://wasm/002d84e2:wasm-function[1305]:0x6d19f
    at wasm://wasm/002d84e2:wasm-function[185]:0x11579
    at wasm://wasm/002d84e2:wasm-function[76]:0x8ab8
    at wasm://wasm/002d84e2:wasm-function[699]:0x530ff
    at wasm://wasm/002d84e2:wasm-function[1792]:0xae0ee

I have a feeling it's something to do with the animarray as i don't seem to get it when I remove this op.

**Comment by steam0r** [comment]
please publish the patch or send a secret url, we cant check invites that are "out of context", sorry
